### DISCUSSION TITLE
A recent spam attack?

### DISCUSSION THREAD

**Discussion by hugary1995** [idaholab/moose#23050] [Q&A General] [answered]
It seems like there was an increase in spam rate over the holidays. A quick google search suggests that the same happened in many repositories. I wonder if we can change some of the moderation settings as suggested here https://github.com/orgs/community/discussions/27247#discussioncomment-4549334

Some unanswered questions got buried as a consequence, unfortunately. So maybe bump your thread if it has slipped through our eyes...

**Comment by GiudGiud** [comment, marked as answer]
Forbidding new github users from using Discussions might be an issue for us. Even at the labs some people have been working exclusively with gitlab/others and just got on github for moose
